examples/python/ColumnOperations/README.md

```diff
@@ -205,13 +208,59 @@ The second function is an aggregation and sums the values in a column. We iterat
         # Yield the row data constructed
         yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
 ```
+### `MaxOfColumns_2` function
+The `MaxOfColumns_2` function computes the maximum in each of two columns and returns the maximum values in two columns, therefore making it appropriate to be used from the Qlik load script. As you can see the function also set the TableDescription header before sending the result.
+
+```python
+    @staticmethod
+    def _max_of_columns_2(request, context):
+        """
+        Find max of each column. This is a table function.
+        :param request: an iterable sequence of RowData
+        :param context:
+        :return: a table with numerical values, two columns and one row
+        """
+
+        result = [_MINFLOAT]*2
+
+        # Iterate over bundled rows
+        for request_rows in request:
+            # Iterating over rows
+            for row in request_rows.rows:
+                # Retrieve the numerical value of each parameter
+                # and update the result variable if it's higher than the previously saved value
+                for i in range(0, len(row.duals)):
+                    result[i]=max(result[i], row.duals[i].numData)
+
+        # Create an iterable of dual with numerical value
+        duals = iter( [SSE.Dual(numData=r) for r in result])
+
+        # Set and send Table header
+        table = SSE.TableDescription(name='MaxOfColumns', numberOfRows=1)
+        table.fields.add(name='Max1', dataType=SSE.NUMERIC)
+        table.fields.add(name='Max2', dataType=SSE.NUMERIC)
+        md = (('qlik-tabledescription-bin', table.SerializeToString()),)
+        context.send_initial_metadata(md)
+        
+        # Yield the row data constructed
+        yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
+```
 
 ## Qlik documents
-An example document is given for Qlik Sense (SSE_Column_Operations.qvf) and QlikView (SSE_Column_Operations.qvw). There are are two sheets in this example. One contains script calls and the other contains user defined function calls. We demonstrate a tensor function, which sums two columns row-wise, and an aggregating function, which sums all rows in a column returning a single value. The data loaded in the Data Load Editor are two fields, *A* and *B*, each of which contain five numeric values.
+An example document is given for Qlik Sense (SSE_Column_Operations.qvf) and QlikView (SSE_Column_Operations.qvw). 
+There are three sheets in this example, one with script calls, one with user defined function calls 
```

Comment:
There are three sheets in this example: one with script calls, one with user defined function calls, and one with the result of an SSE Table Load.